lib/main.js

```diff
@@ -23,18 +23,15 @@ const trivyHelper_1 = require("./trivyHelper");
 const dockleHelper_1 = require("./dockleHelper");
 const inputHelper = __importStar(require("./inputHelper"));
 const download = require('download');
-function getWhitelistFileLoc(whitelistFilePath, whitelistFileBranch) {
+function getWhitelistFileLoc(whitelistFilePath) {
     return __awaiter(this, void 0, void 0, function* () {
-        const whitelistFileUrl = `https://raw.githubusercontent.com/${process.env.GITHUB_REPOSITORY}/${whitelistFileBranch}/${whitelistFilePath}`;
-        const whitelistFilePathParts = whitelistFilePath.split('/');
-        const whitelistFileName = whitelistFilePathParts[whitelistFilePathParts.length - 1];
-        const date = Date.now();
-        const whitelistFileDownloadDir = `${process.env['GITHUB_WORKSPACE']}/_temp/containerScanWhitelist_${date}`;
-        const githubToken = inputHelper.githubToken;
-        console.log(util.format("Downloading whitelist file from %s", whitelistFileUrl));
-        return download(whitelistFileUrl, whitelistFileDownloadDir, { headers: { Authorization: `token ${githubToken}` } }).then(() => {
-            return `${whitelistFileDownloadDir}/${whitelistFileName}`;
-        });
+        const githubWorkspace = process.env['GITHUB_WORKSPACE'];
+        if (!githubWorkspace) {
+            throw new Error("Use actions/checkout action.");
+        }
+        const whitelistFileLoc = githubWorkspace + "/" + whitelistFilePath;

```

Comment:
path should have repo name as well, right?

Reply:
I am assuming it won't have the repo name.